azext_iot/central/_help.py

```diff
@@ -410,10 +413,13 @@ def _load_central_monitors_help():
         "iot central diagnostics monitor-events"
     ] = """
         type: command
-        short-summary: Monitor device telemetry & messages sent to the IoT Hub for an IoT Central app.
+        short-summary: View device telemetry messages sent to the IoT Central app.
         long-summary: |
-                    EXPERIMENTAL requires Python 3.5+
-                    This command relies on and may install dependent Cython package (uamqp) upon first execution.
+                    Shows the telemetry data sent to IoT Central application. By default,
+                    it shows all the data sent by all devices. Use the --device-id parameter
+                    to filter to a specific device.
+                    Requires Python 3.5+. This command relies on and may install dependent
```

Comment:
We don't need the "Requires Python 3.5+" sentence anymore as its implied since we've updated the minimum Python to 3.6 for installation.

Reply:
Removed.